### PR TITLE
Add code samples for Reports API

### PR DESCRIPTION
**Pull request by n-fasano**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/code_samples/PHP-SDK-3.0/reports@events-triggered@{eventType}@rules/get.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getTriggeredEventRuleReport(
+    \Rebilly\Sdk\Model\EventType::ACCOUNT_PASSWORD_RESET_REQUESTED,
```
Comment (ashkarpetin) [inline openapi/code_samples/PHP-SDK-3.0/reports@events-triggered@{eventType}@rules/get.php:4]:
```suggestion
    Rebilly\Sdk\Model\EventType::ACCOUNT_PASSWORD_RESET_REQUESTED,
```

?

File: openapi/paths/data-exports.yaml [hunk 2]
```diff
@@ -45,6 +45,11 @@ post:
     - lang: JavaScript
       source:
         $ref: ../code_samples/JavaScript/data-exports/post.js
+    - lang: PHP
+      label: PHP-SDK-3.0
+      source:
+        $ref: >-
+          ../code_samples/PHP-SDK-3.0/data-exports/post.php
```
Comment (ashkarpetin) [inline openapi/paths/data-exports.yaml:52]:
```suggestion
      source:
        $ref: ../code_samples/PHP-SDK-3.0/data-exports/post.php
```
https://github.com/Rebilly/api-definitions/blob/82fc2268e608a91bb18293a9c92442e76d92a991/openapi/paths/invoices%40%7Bid%7D%40issue.yaml#L48-L49 ?

Comment (ashkarpetin) [reply openapi/paths/data-exports.yaml:52]: Seems formatting is different compared to previous merged PHP-SDK-3.0 samples

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/code_samples/PHP-SDK-3.0/data-exports@{id}/get.php [hunk 3]
```diff
@@ -0,0 +1,3 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->dataExports()->get('dataExportId');
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/data-exports@{id}/get.php:3]:
```suggestion

$dataExport = $service->dataExports()->get('dataExportId');
```

File: openapi/code_samples/PHP-SDK-3.0/histograms@transactions/get.php [hunk 4]
```diff
@@ -0,0 +1,9 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->histograms()->getTransactionHistogramReport(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/histograms@transactions/get.php:3]:
```suggestion

$histogram = $service->histograms()->getTransactionHistogramReport(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@api-log-summary/get.php [hunk 5]
```diff
@@ -0,0 +1,7 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getApiLogSummary(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@api-log-summary/get.php:3]:
```suggestion

$report = $service->reports()->getApiLogSummary(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@cumulative-subscriptions/get.php [hunk 6]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getCumulativeSubscriptions(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@cumulative-subscriptions/get.php:3]:
```suggestion

$report = $service->reports()->getCumulativeSubscriptions(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@dashboard/get.php [hunk 7]
```diff
@@ -0,0 +1,7 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getDashboardMetrics(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@dashboard/get.php:3]:
```suggestion

$dashboard = $service->reports()->getDashboardMetrics(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@dcc-markup/get.php [hunk 8]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getDccMarkup(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@dcc-markup/get.php:3]:
```suggestion

$report = $service->reports()->getDccMarkup(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@disputes/get.php [hunk 9]
```diff
@@ -0,0 +1,7 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getDisputes(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@disputes/get.php:3]:
```suggestion

$report = $service->reports()->getDisputes(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@events-triggered/get.php [hunk 10]
```diff
@@ -0,0 +1,7 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getEventsTriggeredSummary(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@events-triggered/get.php:3]:
```suggestion

$report = $service->reports()->getEventsTriggeredSummary(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@events-triggered@{eventType}@rules/get.php [hunk 11]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getTriggeredEventRuleReport(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@events-triggered@{eventType}@rules/get.php:3]:
```suggestion

$report = $service->reports()->getTriggeredEventRuleReport(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@future-renewals/get.php [hunk 12]
```diff
@@ -0,0 +1,7 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getFutureRenewals(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@future-renewals/get.php:3]:
```suggestion

$report = $service->reports()->getFutureRenewals(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@journal/get.php [hunk 13]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getJournal(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@journal/get.php:3]:
```suggestion

$report = $service->reports()->getJournal(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@kyc-acceptance-summary/get.php [hunk 14]
```diff
@@ -0,0 +1,6 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getKycAcceptanceSummary(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@kyc-acceptance-summary/get.php:3]:
```suggestion

$report = $service->reports()->getKycAcceptanceSummary(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@kyc-rejection-summary/get.php [hunk 15]
```diff
@@ -0,0 +1,6 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getKycRejectionSummary(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@kyc-rejection-summary/get.php:3]:
```suggestion

$report = $service->reports()->getKycRejectionSummary(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@kyc-request-summary/get.php [hunk 16]
```diff
@@ -0,0 +1,6 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getKycRequestSummary(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@kyc-request-summary/get.php:3]:
```suggestion

$report = $service->reports()->getKycRequestSummary(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@monthly-recurring-revenue/get.php [hunk 17]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getMonthlyRecurringRevenue(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@monthly-recurring-revenue/get.php:3]:
```suggestion

$report = $service->reports()->getMonthlyRecurringRevenue(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@renewal-sales/get.php [hunk 18]
```diff
@@ -0,0 +1,7 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getRenewalSales(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@renewal-sales/get.php:3]:
```suggestion

$report = $service->reports()->getRenewalSales(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@retention-percentage/get.php [hunk 19]
```diff
@@ -0,0 +1,9 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getRetentionPercentage(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@retention-percentage/get.php:3]:
```suggestion

$report = $service->reports()->getRetentionPercentage(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@retention-value/get.php [hunk 20]
```diff
@@ -0,0 +1,9 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getRetentionValue(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@retention-value/get.php:3]:
```suggestion

$report = $service->reports()->getRetentionValue(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@revenue-audit/get.php [hunk 21]
```diff
@@ -0,0 +1,3 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getRevenueAudit(limit: 5);
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@revenue-audit/get.php:3]:
```suggestion

$report = $service->reports()->getRevenueAudit(limit: 5);
```

File: openapi/code_samples/PHP-SDK-3.0/reports@revenue-waterfall/get.php [hunk 22]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getRevenueWaterfall(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@revenue-waterfall/get.php:3]:
```suggestion

$report = $service->reports()->getRevenueWaterfall(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@subscription-cancellation/get.php [hunk 23]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getSubscriptionCancellation(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@subscription-cancellation/get.php:3]:
```suggestion

$report = $service->reports()->getSubscriptionCancellation(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@subscription-renewal/get.php [hunk 24]
```diff
@@ -0,0 +1,7 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getSubscriptionRenewal(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@subscription-renewal/get.php:3]:
```suggestion

$report = $service->reports()->getSubscriptionRenewal(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@time-series-transaction/get.php [hunk 25]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getTimeSeriesTransaction(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@time-series-transaction/get.php:3]:
```suggestion

$report = $service->reports()->getTimeSeriesTransaction(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@transactions-time-dispute/get.php [hunk 26]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getTransactionsTimeDispute(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@transactions-time-dispute/get.php:3]:
```suggestion

$report = $service->reports()->getTransactionsTimeDispute(
```

File: openapi/code_samples/PHP-SDK-3.0/reports@transactions/get.php [hunk 27]
```diff
@@ -0,0 +1,8 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->reports()->getTransactions(
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/reports@transactions/get.php:3]:
```suggestion

$report = $service->reports()->getTransactions(
```

File: openapi/code_samples/PHP-SDK-3.0/subscriptions@{subscriptionId}@summary-metrics/get.php [hunk 28]
```diff
@@ -0,0 +1,3 @@
+$service = new Rebilly\Sdk\ReportsService($client);
+
+$service->subscriptions()->getSubscriptionSummaryMetrics('subscriptionId');
```
Comment (justnero) [inline openapi/code_samples/PHP-SDK-3.0/subscriptions@{subscriptionId}@summary-metrics/get.php:3]:
```suggestion

$report = $service->subscriptions()->getSubscriptionSummaryMetrics('subscriptionId');
```

**Comment by n-fasano** [comment]
@Rebilly/php please re-review

**Review by ashkarpetin — APPROVED** [review]

**Review by justnero — APPROVED** [review]

**Pull request merged**